### PR TITLE
Communication Section

### PR DESCRIPTION
**Pull request by PaigeCD**
This is Group 3's addition to the Communication section of the Code of Conduct. We're hoping to merge this so others who wish to add to this section can do so.

### CODE REVIEW

File: conduct.md
Comment (noorbuchi) [inline conduct.md]: Extra space is not needed between list items, please remove this to stay consistent with other sections

File: conduct.md
Comment (noorbuchi) [inline conduct.md]: No extra space is needed here either

File: conduct.md
Comment (noorbuchi) [inline conduct.md]: Please make sure to communicate these changes to the Deadlines group, they are responsible for this section and they should be okay with your additions.

File: conduct.md
Comment (noorbuchi) [inline conduct.md]: Extra space is not needed here either

File: conduct.md
Comment (noorbuchi) [inline conduct.md]: I think it's good to be specific here. What method(s) of communication should the team use? Slack, GitHub Issue Trackers, email, all of them?

File: conduct.md
Comment (noorbuchi) [inline conduct.md]: While I definitely think that making fun of other peoples' contributions is unacceptable, I don't think this point belongs in the communication section. You could also combine it with the inappropriate use of communication platforms and identify how that looks like.

File: conduct.md
Comment (noorbuchi) [inline conduct.md]: Extra space is not needed between list items, please remove this to stay consistent with other sections

File: conduct.md
Comment (noorbuchi) [inline conduct.md]: Another extra space

File: conduct.md
Comment (noorbuchi) [inline conduct.md]: Extra empty line should be removed

File: conduct.md
Comment (noorbuchi) [inline conduct.md]: Please remove empty lines from this section

File: conduct.md
Comment (noorbuchi) [inline conduct.md]: Empty line

File: conduct.md
Comment (noorbuchi) [inline conduct.md]: Please let the group responsible for postmortems know about these changes. Additionally, you should keep the `TODO` at the bottom of the list until the group responsible finalizes their contribution

File: conduct.md
Comment (noorbuchi) [inline conduct.md]: Can you please explain the purpose of this change. It could potentially introduce unnecessary inconsistency with other branches.

File: conduct.md
Comment (noorbuchi) [inline conduct.md]: Please make sure to add the names of all group members. Additionally, the `TODO` section should be kept until other groups have also added their names

**Review by noorbuchi — CHANGES_REQUESTED** [review]
Hello! Thank you for your work on this PR, I added some change requests that should be addressed before merging this PR. Can you also please explain why all bullet points in the document were changed from `*` to `-`? Both of these notations serve the same purpose. Changing them can create unnecessary conflicts so I strongly encourage you to address that.

**Comment by noorbuchi** [comment]
It's also not advisable to have someone who worked on the PR also be a reviewer. So please make sure to request the review of someone who hasn't contributed to this specific PR, it should preferably be a TL

**Review by PaigeCD — COMMENTED** [review]
In this commit I attempted to address a few concerns raised by Noor. In this commit I:
- Removed many empty lines
- Temporarily removed additions (with permission) to other sections so that they can be reviewed later
- Changed all - to * for consistency with main
- Made a specification in Communication requested by Noor
- Added punctuation to to Communication for consistency
- Put the TODO for adding names back
- Added my name to the file

File: conduct.md
Comment (noorbuchi) [reply conduct.md]: Please don't forget to address this request

File: conduct.md
Comment (noorbuchi) [inline conduct.md]: I believe that there is a missing name that contributed to this PR but has not been added to the list

**Review by noorbuchi — CHANGES_REQUESTED** [review]
Thank you for addressing the changes I requested! Please feel free to resolve the comments if the changes were implemented. I also made a couple of other requests in this review. Finally, there are merge conflicts between this and the main branch that need to be resolved before merging this PR.

**Comment by corlettim** [comment]
Great work so far! After taking a look at Noor's comments, please make sure you resolve the merge conflicts.

**Review by corlettim — CHANGES_REQUESTED** [review]
Great work so far! After taking a look at Noor's comments, please make sure you resolve the merge conflicts.

**Review by noorbuchi — DISMISSED** [review]
I think this looks good! You've addressed all the requested changes. Please wait until this PR gets the needed approvals before merging.

**Review by noorbuchi — DISMISSED** [review]

**Review by corlettim — DISMISSED** [review]

File: conduct.md
Comment (PaigeCD) [inline conduct.md]: These extra spaces need removed again

**Review by PaigeCD — COMMENTED** [review]
It looks like you may have forgotten to pull before pushing this and some things have gotten undone.

**Review by enpuyou — CHANGES_REQUESTED** [review]
@PaigeCD It seems like the section your group contributed still has some markdown linting errors raised in Github Action. Make sure to resolve those before the PR could be merged.

File: conduct.md
Comment (enpuyou) [inline conduct.md]: I think it would be best to preserve the original text for Level II, while having what your group has proposed here for the action.

**Review by enpuyou — DISMISSED** [review]

**Review by corlettim — CHANGES_REQUESTED** [review]
Everything is coming together really well! Please make sure to fix the linting errors that you are now facing

**Review by jjumadinova — DISMISSED** [review]

File: conduct.md
Comment (corlettim) [inline conduct.md]: Make sure to remove your TODO statements

File: conduct.md
Comment (corlettim) [inline conduct.md]: Make sure to remove this TODO

File: conduct.md
Comment (corlettim) [inline conduct.md]: remove the TODO please :)

**Review by corlettim — CHANGES_REQUESTED** [review]

File: conduct.md
Comment (corlettim) [inline conduct.md]: I think you meant to put this under the shall not section but it is showing up in the shall section. I may be seeing it wrong but double check this is in the right spot.

**Review by corlettim — CHANGES_REQUESTED** [review]

File: conduct.md
Comment (corlettim) [inline conduct.md]: I think this needs be reworded. It is a little repetitive with "classmates" and "outside classmates"

**Review by corlettim — CHANGES_REQUESTED** [review]

**Comment by corlettim** [comment]
Please make sure to resolve the conflicts with main branch and then you should be good to go

**Review by enpuyou — DISMISSED** [review]
Thanks for addressing all the raised issues! I think the PR is good to go.

**Review by corlettim — DISMISSED** [review]

**Review by noorbuchi — DISMISSED** [review]
Everything looks good! Please resolve merge conflicts with `main` and we should be ready to merge your PR.

File: conduct.md
Comment (noorbuchi) [inline conduct.md]: I think these should be in the unacceptable conduct section instead

Comment (corlettim) [inline conduct.md]: I think these 3 points need to be under "shall not" instead of "shall".

**Review by corlettim — CHANGES_REQUESTED** [review]

File: conduct.md
Comment (noorbuchi) [inline conduct.md]: I think that `Teamwork` is a level above `Participation`, `Decisions`, etc.. So the text under it describes this whole section related to teamwork. You can either remove this or incorporate parts of it in the other `Team Work` sub-sections.

**Review by noorbuchi — CHANGES_REQUESTED** [review]
Overall everything looks great! I just noticed two small issues that should be addressed.

File: conduct.md
Comment (noorbuchi) [reply conduct.md]: @corlettim  addressed this in the comment above so I will resolve mine

**Review by enpuyou — APPROVED** [review]

**Review by noorbuchi — APPROVED** [review]
Looks good!

**Review by jjumadinova — APPROVED** [review]

**Pull request merged**
